### ISSUE TITLE
Author names can be exposed to some reviewers

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#3070]
If a reviewer also has an author role, they will be shown the author's name in the submission list.

Reported: https://forum.pkp.sfu.ca/t/ojs-3-1-reviewer-can-see-author-name-after-review-complete/35344/10

**Comment by NateWr** [comment]
@asmecher PRS:
https://github.com/pkp/pkp-lib/pull/3076
https://github.com/pkp/ojs/pull/1705
https://github.com/pkp/omp/pull/471

The OJS/OMP PRs are just submodule updates.

**Comment by NateWr** [comment]
Thanks for the code review, @asmecher. Just the one query left on [when to consider something published](https://github.com/pkp/pkp-lib/pull/3076#discussion_r152238299).

**Comment by NateWr** [comment]
Ok, I've updated the checks on whether a submission is published for OJS and OMP. Can you take another look at the PRs @asmecher? (There's some whitespace noise in the OMP one, but the relevant changes are in https://github.com/pkp/omp/pull/471/commits/d2ee53107ba4695de23d13f58a8edb5e223b8406).

**Comment by asmecher** [comment]
Thanks, @NateWr, just a couple of comments -- please go ahead and merge when ready. Don't forget to back-port to `ojs-stable-3_1_0` as well.

**Comment by NateWr** [comment]
Ok, just waiting on tests from PRs above to be fixed.

PRs for `ojs-stable-3_1_0`:
https://github.com/pkp/pkp-lib/pull/3117
https://github.com/pkp/ojs/pull/1727

**Comment by NateWr** [comment]
@asmecher All merged into `master` and `ojs-stable-3_1_0`.

**Comment by asmecher** [comment]
Excellent, thanks!
